web-nfc/NFCReader_options_recordType_empty-manual.https.html

```diff
@@ -22,13 +22,10 @@
 setup({ explicit_timeout: true });
 
 const desc = "Test that write and read data succeed when recordType is set to 'empty'.";
-const readOptions = {
-  recordType: "empty",
-  mediaType: "",
-  url: ""
-}
+const readOptions = {recordType: "empty"};
+const invalidReadOptions = {recordType: "json"};
```

Comment:
"invalidReadOptions" sounds odd, as it is not an invalid one.
How about "unacceptableReadOptions"?